### PR TITLE
Szymon statistics unit tests

### PR DESCRIPTION
**Pull request by SzymonB5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/preset-typescript": "^7.21.4",
```
Comment (kasia251011) [inline package.json:57]: You dont have to install anything, this template from crate-react-app is already prepared for jest.

File: package.json [hunk 2]
```diff
@@ -63,6 +64,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "jest": "^27.5.1",
```
Comment (kasia251011) [inline package.json:67]: You dont have to install anything, this template from crate-react-app is already prepared for jest.

File: src/utilities/BasicStatisticsFunctions.test.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
+import {describe, expect, test} from '@jest/globals';
```
Comment (kasia251011) [inline src/utilities/BasicStatisticsFunctions.test.tsx:1]: This import is uneccesary

File: {
Comment (kasia251011) [inline {:1]: unused file.

**Review by kasia251011 — CHANGES_REQUESTED** [review]
Tests are fine :) Just repair build problem, and delete unnecessary libraries.

**Review by kasia251011 — APPROVED** [review]

**Review by cpprian — APPROVED** [review]

**Pull request merged**